src/components/ContributionsEpic.tsx

```diff
@@ -107,6 +107,9 @@ export type EpicTargeting = {
     isMinuteArticle: boolean;
     isPaidContent: boolean;
     tags: Tag[];
+
+    isRecurringContributor: boolean;
+    lastOneOffContributionDate?: string; // TODO possibly use a number or date format here
```

Comment:
Just highlighting the `TODO` here. Should we resolve this question before merging?

Reply:
I think `lastOneOffContributionDate` is never taken out of the POST payload in `server.tsx` so won't be available here.